src/themes/default/components/core/blocks/Cms/Block.vue

```diff
@@ -26,20 +26,7 @@ export default {
       required: false
     }
   },
-  // asyncData ({ store, route, context }) {
-  // @TODO to cover SSR need find a way to pass props identifier/id to the asyncData()
-  // for now it is not possible but assuming from some info it could be available later
-  // we try to dispatch 'single' action which actually dispatch list to prefetch data anyway
-
-  /* return new Promise((resolve, reject) => {
-    store.dispatch('cmsBlock/single', {
-      value: this.identifier
-    }).then(res => {
-      return resolve(res)
-    })
-  }) */
-  // },
-  created () {
```

Comment:
You should keep the created method. It´s needed for CSR. Otherwise blocks won´t be fetched when you browse the page. 
What you can do in the created method, you can prevent to execute created hook when SSR. 
if (!isServer) {... content of created method}
So best would be to add another method, to be used by **created** and **serverPrefetch**

Reply:
Good point! serverPrefetch is not triggered on client side. Thanks! :)